src/pages/ArticleDetailPage.tsx

```diff
@@ -40,22 +41,23 @@ const ArticleDetailPage = () => {
 
   const toggleLikeMutate = () => {
     if (myLike) {
-      likeDeleteMutate(myLike._id, {
-        onSuccess: (newLike) =>
+      likeDeleteMutate(myLike._id);
+    } else {
+      likeCreateMutate(_id, {
+        onSuccess: (newLike) => {
```

Comment:
```suggestion
myLike
  ? likeDeleteMutate(myLike._id)
  : likeCreateMutate(_id, {
    onSuccess: (newLike) => { 
      likeNotificationMutate({
            notificationType: 'LIKE',
            notificationTypeId: newLike._id,
            userId: postUserId,
            postId: newLike.post,
          });
        }
      })
```

p5: 삼항 연산자를 사용해보는 건 어떨까요? (괄호가 맞는지는 헷갈리네요....)

Reply:
Onsuccess부분은 mutate로직에서 받아도될것같아용! 취향차이이긴하지만욥..페이지가 조금 지저분해보일수있을것같아용??

Reply:
저도 처음엔 따로 빼려고 했는데 게시물 유저 아이디를 받아야 하길래... 여기다가 했습니다!! postUserId를 전달해주는 방법이 뭘까여??